crates/wasi-component-loader/src/extension/api/since_0_19_0/wit/mod.rs

```diff
@@ -0,0 +1,34 @@
+pub mod authorization_types;
```

Comment:
a lot of copy pasted files, not the end of the world but would be better to just reference the existing ones. like authoriation types aren't new since a while